### PR TITLE
Add fronts-banners to section fronts when in `sectionFrontsBannerAds` test

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Add banner slots to section fronts on desktop when in the `sectionFrontsBannerAds` test.

There are a few tweaks to the network front test code to accommodate the section front test.

Including renaming the current `frontsBanner` variables to `networkFrontsBanner` to better distinguish the tests.

And use the alternative name for sections which is collections to reduce confusion with section fronts and the sections/collections on the fronts.

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| ![before][] | ![after][] |

[before]: https://github.com/guardian/dotcom-rendering/assets/1731150/683d8257-44d7-4bfa-93ff-68e4eee9e1b5
[after]: https://github.com/guardian/dotcom-rendering/assets/1731150/65e8a3b0-4928-4aba-b623-217e2d484a5a

### Examples of spacing I've gone for

| /uk/sport    | /uk/commentisfree      |
| ----------- | ---------- |
| ![sport][] | ![comment][] |

[sport]: https://github.com/guardian/dotcom-rendering/assets/1731150/bcff2ee5-9626-4ff6-a921-1b8306a219d9

[comment]: https://github.com/guardian/dotcom-rendering/assets/1731150/f8d81fea-38ad-49f1-8942-677fa6c06ad2

### CODE REVIEW

**Review by chrislomaxjones — APPROVED** [review]
LGTM 👍

File: dotcom-rendering/src/layouts/FrontLayout.tsx [hunk 1]
```diff
@@ -81,20 +84,21 @@ const isToggleable = (
...
 	const minContainers = isPaidContent ? 1 : 2;
 	if (
 		collectionCount > minContainers &&
 		index === getMerchHighPosition(collectionCount) &&
-		!(isInFrontsBannerTest && isNetworkFront)
+		!isInNetworkFrontsBannerTest &&
+		!isInSectionFrontsBannerTest
 	) {
 		return (
 			<AdSlot
```
Comment (domlander) [inline dotcom-rendering/src/layouts/FrontLayout.tsx:104]: Users in the test but not on a front that is in the test, will not get the `merchandising-high` slots. I think we need the check to be something like:

`!(isInNetworkFrontsBannerTest && isFrontInTheNetworkTest) &&`
`!(isInSectionFrontsBannerTest && isFrontInTheSectionTest)`

File: dotcom-rendering/src/layouts/FrontLayout.tsx [hunk 2]
```diff
@@ -125,18 +129,14 @@ export const decideAdSlot = (
 export const decideFrontsBannerAdSlot = (
```
Comment (domlander) [inline dotcom-rendering/src/layouts/FrontLayout.tsx:129]: The comment is out of date. Can it be updated?

File: dotcom-rendering/src/layouts/FrontLayout.tsx [hunk 3]
```diff
@@ -205,14 +205,26 @@ export const FrontLayout = ({ front, NAV }: Props) => {
...
-	// For the FrontsBannerAds test, we're not using contentType === "Network Front",
-	// just in case Europe goes live before the testing concludes.
-	// We don't want Europe in this test.
-	const networkFrontPageIds = ['uk', 'us', 'au', 'international'];
-	const isInFrontsBannerTest =
+	const isInNetworkFrontsBannerTest =
 		!!switches.frontsBannerAdsDcr &&
 		abTests.frontsBannerAdsDcrVariant === 'variant' &&
-		networkFrontPageIds.includes(front.config.pageId);
+		Object.keys(networkFrontsBannerAdCollections).includes(
```
Comment (domlander) [inline dotcom-rendering/src/layouts/FrontLayout.tsx:211]: Much better!

**Pull request merged**